js/script.js

```diff
@@ -0,0 +1,8 @@
+const addToShopping = document.querySelectorAll(".pizza__addToShopping");
+
```

Comment:
congratulations!!! it's your first js script

Reply:
+

Reply:
ok